app/src/main/java/com/univoice/feature/home/HomeFragment.kt

```diff
@@ -1,88 +1,502 @@
+    ) {
+        Image(
+            imageVector = ImageVector.vectorResource(id = R.drawable.ic_appbar_logo),
+            contentDescription = stringResource(R.string.description_home_logo_img),
+            modifier = Modifier.padding(start = 16.dp)
+        )
+    }
+    Spacer(modifier = Modifier.padding(bottom = 10.dp))
+    Text(
+        text = stringResource(R.string.text_home_quick_scan_title),
+        style = head5Bold,
+        color = Font_B01,
+        modifier = Modifier.padding(start = 16.dp)
+    )
+    Spacer(modifier = Modifier.padding(bottom = 11.dp))
+    LazyRow(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight(),
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        contentPadding = PaddingValues(horizontal = 4.dp, vertical = 6.dp),
+    ) {
+        items(
+            items = homeViewModel.mockQuickScanList,
+            key = { data -> data.name },
+        ) { data ->
+            Column(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .width(95.dp)
+                    .clickable {
+                        // To-do : 추가해야 함
+                    },
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.Center
+            ) {
+                HomeQuickScanItem(data)
+            }
+        }
+    }
+    Spacer(modifier = Modifier.padding(bottom = 8.dp))
+}
+
+@Composable
+fun HomeQuickScanItem(data: QuickScanListEntity) {
+    Box {
+        Image(
+            imageVector = ImageVector.vectorResource(id = R.drawable.ic_launcher_background),
+            contentDescription = stringResource(R.string.text_home_quick_scan_title),
+            contentScale = ContentScale.Crop,
+            modifier = Modifier
+                .size(68.dp)
+                .aspectRatio(1f)
+                .clip(CircleShape)
+        )
+        Surface(
+            color = Blue300,
+            shape = CircleShape,
+            modifier = if (data.count.toString().length == 1) {
+                Modifier
+                    .align(Alignment.TopEnd)
+                    .offset(x = (-4.5).dp, y = (-6).dp)
+            } else {
+                Modifier
+                    .align(Alignment.TopEnd)
+                    .offset(x = 2.5.dp, y = (-6).dp)
+            }
+        ) {
+            Box(
+                contentAlignment = Alignment.Center,
+                modifier = Modifier.wrapContentSize()
+            ) {
+                Text(
+                    text = data.count.toString(),
+                    style = button2Semi,
+                    color = Font_W01,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(
+                        vertical = 2.dp,
+                        horizontal = 6.dp
+                    ),
+                )
+            }
```

Comment:
요롷게 Box로 묶지 않고 바로 Text 적용해도 됩니다ㅏㅏ!!

```suggestion
                Text(
                    text = data.count.toString(),
                    style = button2Semi,
                    color = Font_W01,
                    textAlign = TextAlign.Center,
                    modifier = Modifier.padding(
                        vertical = 2.dp,
                        horizontal = 6.dp
                    ),
                )
```

Reply:
그러네욤! 예리하다!